File: Facultate/Anul 2/Sem 2/MPP/Laburi/Lab5/CSharpTema3/CSharpLab3/dtos/RideDTO.cs
namespace CSharpLab3.dtos;

public class RideDTO
{
    public int SeatNo { get; }
    public string ClientName { get; }

    public RideDTO(int seatNo, string clientName)
    {
        SeatNo = seatNo;
        ClientName = clientName;
    }
}

File: Facultate/Anul 2/Sem 2/MPP/Laburi/JavaTema4/JavaTema4/Shared/src/main/java/dtos/RideDTO.java
package dtos;

public class RideDTO {

    private final int noSeats;
    private final String reservedBy;

    public RideDTO(int noSeats, String reservedBy) {
        this.noSeats = noSeats;
        this.reservedBy = reservedBy;
    }

    public int getNoSeats() {
        return noSeats;
    }

    public String getReservedBy() {
        return reservedBy;
    }

}
